src/components/vote/Vote.jsx

```diff
@@ -0,0 +1,48 @@
+import axios from "axios";
+import React, { useState } from "react";
+import "./vote.css";
+
+const Vote = ({ article }) => {
+  const [toggle, setToggle] = useState(false);
+  const [votes, setVotes] = useState(article.votes);
+
+  const handleClick = () => {
+    setToggle(!toggle);
```

Comment:
You'd be better off here using the current value of toggle, so `setToggle((currentToggle) => !currentToggle)`